integrationtests/src/main/java/io/kroxylicious/proxy/testkafkacluster/KeytoolCertificateGenerator.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy.testkafkacluster;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.IOException;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.GeneralSecurityException;
+import java.security.KeyStore;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KeytoolCertificateGenerator {
+    private static String password;
+    private static final String certsDirectory = "/tmp/kproxy";
+    private static final Path certFilePath = Paths.get(certsDirectory, "kafka.jks");
```

Comment:
This is _better_, but I think it still precludes running tests in parallel without having conflicts. 

To make that work we'd need to step away from the static methods and just use `Files.createTempDirectory` in a constructor, making all the current static methods into instance methods.

Reply:
You are right, I didn't think about parallelism and that's right, we have to avoid static methods. I will do this change.

Reply:
Instead of `Files.createTempDirectory` maybe use JUnits [TempDir annotation](https://junit.org/junit5/docs/5.4.1/api/org/junit/jupiter/api/io/TempDir.html)